WalletWasabi.Fluent/ViewModels/AddWallet/RecoverWalletViewModel.cs

```diff
@@ -77,6 +77,7 @@ private async Task OnNextAsync(string? walletName)
 							password!,
 							Services.WalletManager.Network,
 							AccountKeyPath,
+							null,
```

Comment:
No, it is exactly the opposite! We have to remove the keypaths from the recovery process because that's stupid. That's what we have to do next: update the UI to remove all those things like keypaths, gaps, etc and update others like minlimit, output descrip´tors, etc

Reply:
Added here: https://github.com/zkSNACKs/WalletWasabi/issues/9622